releasenotes/notes/Add-DogStatsD-metrics-to-count-metric-types.-4d74421bcd2222e6.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    DogStatsD emits a new metric `metric_type_count` that tracks gauge, counters etc totals. Exposed via COATs as well as contexts_by_mtype.
```

Comment:
Changelog is not markdown, but RST and uses double backticks for verbatim text.

However, do we need to highlight COAT changes to users?

```suggestion
    DogStatsD emits a new metric ``metric_type_count`` that tracks gauge, counters etc totals. Exposed via COATs as well as ``contexts_by_mtype``.
```

Reply:
> However, do we need to highlight COAT changes to users?

➕ My understanding is that the changelog is user-facing and so we should describe changes in terms of the user-facing impact. In this case, I think the second sentence would not be understandable to users. We could drop it or rewrite it to say that the metric is published as internal telemetry for future improvements.

Reply:
Resolved in [57fcdec](https://github.com/DataDog/datadog-agent/pull/39944/commits/57fcdec4681b7e3ba5bb973abb762b93c6e3a041).